src/logic/basic.lean

```diff
@@ -155,7 +155,10 @@ assume ⟨h⟩, h.elim
 lemma congr_arg_heq {α} {β : α → Sort*} (f : ∀ a, β a) : ∀ {a₁ a₂ : α}, a₁ = a₂ → f a₁ == f a₂
 | a _ rfl := heq.rfl
 
-lemma plift.down_inj {α : Sort*} : ∀ (a b : plift α), a.down = b.down → a = b
+lemma ulift.down_injective {α : Sort*} : function.injective (@ulift.down α)
```

Comment:
Ooh, but I think you should also keep the old version, and mark it `@[simp]`.

Reply:
How should the old versions be called?

Reply:
Let me be more precise, because I actually meant: old name, old statement, except `↔` instead of `→`, and then mark it `@[simp]`.
So the following statement (and similar for `ulift`):
```
@[simp] lemma plift.down_inj {α : Sort*} : ∀ (a b : plift α), a.down = b.down ↔ a = b
```